app/game/game.service.js

```diff
@@ -452,11 +452,31 @@ module.exports = (db) => {
         }));
     };
 
+    const getPlayerStatsByWeek = async (week, playerId) => {
+        const stats = await db.any(`
+            SELECT gps.*
+            FROM game_player_stat gps
```

Comment:
Recommend refactoring this SQL query so that `game` is the top-level table in the query as this will greatly impact query performance. For example,

```
SELECT gps.athlete_id, ...
FROM game AS g
    INNER JOIN ...
```

As a general principle in SQL, it is always optimal to SELECT from the narrowest relation first as clamps down on the number of rows being returned at the top and then joined upon.